plugins/gcloud_image_tagger/lib/gcloud_image_tagger/samson_plugin.rb

```diff
@@ -12,13 +12,16 @@ def tag(deploy)
         return unless builds = deploy.project.builds.
           where(git_sha: deploy.job.commit).where.not(docker_repo_digest: nil).to_a.presence
 
+        gcloud_options = ENV.fetch('GCLOUD_IMG_TAGGER_OPTS', '').split(' ')
+
         builds.each do |build|
           digest = build.docker_repo_digest
           next unless digest =~ /(^|\/|\.)gcr.io\// # gcr.io or https://gcr.io or region like asia.gcr.io
           base = digest.split('@').first
           tag = deploy.stage.permalink
           beta = container_in_beta
-          command = ["gcloud", *beta, "container", "images", "add-tag", digest, "#{base}:#{tag}", "--quiet"]
+          command = ["gcloud", *beta, "container", "images", "add-tag", digest, "#{base}:#{tag}",
+                     "--quiet", *gcloud_options]
```

Comment:
2-space indent plz

Reply:
I tried changing it to a 2-space indent, and now rubocop is complaining.

```
plugins/gcloud_image_tagger/lib/gcloud_image_tagger/samson_plugin.rb:25:13: C: Style/AlignArray: Align the elements of an array literal if they span more than one line.
            "--quiet", *gcloud_options]
            ^^^^^^^^^
```